src/main/java/net/snowflake/ingest/streaming/internal/OpenChannelResponse.java

```diff
@@ -130,4 +131,13 @@ void setEncryptionKeyId(Long encryptionKeyId) {
   Long getEncryptionKeyId() {
     return this.encryptionKeyId;
   }
+
+  @JsonProperty("stage_location")
```

Comment:
Lets add this when adding External volume logic to main? Remove for now?

Reply:
Sure, removed.